### PR TITLE
Static layout with dummy data

### PR DESCRIPTION
**Pull request by KanekarSnehal**
created static layout with dummy data
created modal for editing and creating new note
created color palette and dropdown list for labels.

Files to check for:
- src/components/Dropdown
- src/components/Header
- src/components/HomePageFilter
- src/components/ModalInput
- src/components/NoteCards
- src/components/Palette
- src/components/SideNavigation
- src/components/UpdateLabels
- src/context/ModalContext
- src/Pages/ArchivePage
- src/Pages/LabelPage
- src/Pages/TrashPage

### CODE REVIEW

File: src/components/SideNavigation/SideNavigation.jsx [hunk 1]
```diff
@@ -1,34 +1,84 @@
 import "./sidenavigation.css";
+import { useNavigate } from "react-router-dom";
+import { useModal, useAuthContext } from "../../context/index";
+import { ModalInput } from "../ModalInput/ModalInput";
+
 export const SideNavigation = () => {
+  const { modalState, modalDispatch } = useModal();
+  const { user } = useAuthContext();
+  console.log(user);
```
Comment (prathmeshjagtap) [inline src/components/SideNavigation/SideNavigation.jsx:9]: you  can remove the console.log

**Review by prathmeshjagtap — APPROVED** [review]
code is well structured and Clean , just a suggestion add a Preview Link or any video

File: src/components/SideNavigation/SideNavigation.jsx (hunk 1 above)
Comment (KanekarSnehal) [reply src/components/SideNavigation/SideNavigation.jsx:9]: done

File: src/components/ModalInput/ModalInput.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+export const ModalInput = () => {
+  const tagOptions = ["class", "work", "study"];
+  const priorityOptions = ["high", "medium", "low"];
+  const { modalState, modalDispatch } = useModal();
+  const { noteInfo, buttonType } = modalState;
+
+  return (
+    <div
+      className="overlay"
+      onClick={() => modalDispatch({ type: "CLOSE_MODAL", payload: "" })}
```
Comment (ajay-mandaviya) [inline src/components/ModalInput/ModalInput.jsx:14]: if u don't want to pass payload then remove it from here

File: src/components/ModalInput/ModalInput.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            <i className="far fa-trash-alt mr-16 icon"></i>
+            <Dropdown data={tagOptions} />
+            <Dropdown data={priorityOptions} />
+
+            <div className="ml-auto">
+              <button className="btn primary-btn">{buttonType}</button>
+              <button
+                className="btn secondary-btn"
+                onClick={() =>
+                  modalDispatch({ type: "CLOSE_MODAL", payload: "" })
```
Comment (ajay-mandaviya) [inline src/components/ModalInput/ModalInput.jsx:55]: same here remove payload if u don't pass it anything

File: src/components/NoteCards/NoteCards.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+              <div className="input-text-section">
+                <h6>{dataItem.title}</h6>
+                <p>{dataItem.body}</p>
+              </div>
+              <button className=" note-pin badge-top-right ">
+                <i className="fas fa-thumbtack icon"></i>
+              </button>
+              <div className="label-container">
+                {dataItem.label !== [] &&
+                  dataItem.label.map((labelItem) => (
```
Comment (ajay-mandaviya) [inline src/components/NoteCards/NoteCards.jsx:23]: use dataItem.length === 0

**Review by ajay-mandaviya — APPROVED** [review]
Apart from  comments  ,  everything is well and proper structure

File: src/components/ModalInput/ModalInput.jsx (hunk 3 above)
Comment (KanekarSnehal) [reply src/components/ModalInput/ModalInput.jsx:55]: Made the changes thank you for pointing out.

File: src/components/NoteCards/NoteCards.jsx (hunk 4 above)
Comment (KanekarSnehal) [reply src/components/NoteCards/NoteCards.jsx:23]: done

**Pull request merged**